readme.md

```diff
@@ -130,6 +130,15 @@
 | ------: | -------: | ------: |
 |   3 - 6 |       10 | 60-120m |
 
+
+##[Monopoly] (https://en.m.wikipedia.org/wiki/File:Monopoly_pack_logo.png)
+
+Monopoly is a board game where players roll two six-sided dice to move around the game board, buying and trading properties, and developing them with houses and hotels. 
```

Comment:
Per [the formatting guide](https://github.com/edm00se/awesome-board-games/blob/master/formatting.md#example), the description should be contained within a "quote" block, making use of the `>` character at the front of the line.

```suggestion
> Monopoly is a board game where players roll two six-sided dice to move around the game board, buying and trading properties, and developing them with houses and hotels. 
```